src/fluxbox.cc

```diff
@@ -514,7 +514,7 @@ void Fluxbox::initScreen(BScreen *screen) {
     // initiate atomhandler for screen specific stuff
     STLUtil::forAll(m_atomhandler, 
             CallMemFunWithRefArg<AtomHandler, BScreen&, void>(&AtomHandler::initForScreen, *screen));
-    //STLUtil::forAll(m_atomhandler, bind2nd(mem_fun(&AtomHandler::initForScreen), *screen));
+    //STLUtil::forAll(m_atomhandler, bind2nd(mem_fn(&AtomHandler::initForScreen), *screen));
```

Comment:
stale commented bind2nd, only mem_fun replaced

Reply:
My apologies for not fully understanding: Is this comment a request for a change to this PR? Happy to make the desired change if you can let me know what you would like. It is true that because of a search-and-replace, this change was made in a commented-out line, but it is consistent with the operative changes and didn't seem harmful, as the line is commented out anyway. Thanks for letting me know your preference here.

Reply:
I mostly wanted to sign that "yup, I've read that" and this is the only thing to add to it from here.

The "problem" isn't the change in the commented code - that's fine.
The "problem" is that you fixed the mem_fun reference but the bin2nd patter remains so if anyone ever figures to uncomment that line, and isolated warning (or by that time error) will show up here.
I'd suggest to change the bind2nd in the context this commit as weill (though inert because of the comment)

Reply:
Done, thanks for clarifying.